src/lib/util/util.ts

```diff
@@ -403,8 +403,15 @@ export function getImage(message: Message): string | null {
 	return attachment ? attachment.proxyURL || attachment.url : null;
 }
 
+export async function fetchColor(message: Message) {
+	const { userProfiles } = await DbSet.connect();
+	const user = await userProfiles.findOne(message.author.id);
+
+	return user?.color || message.member?.displayColor || BrandingColors.Primary;
+}
```

Comment:
This should get a unit test. Mock out a message:
```ts
const fakeMessage = {
  author: {
    id: '132456978'
  }
}
```

And mock `DbSet.connect()`: `jest.spyOn(DbSet, 'connect').mockImplementation(() => fakeUserProfiles)` (create a fakeUserProfiles constant which has `findOne` method)

Then ensure all branches are covered.

Reply:
Otherwise we can't merge because coveralls makes CI red (and rightfully so)

Reply:
This code is outdated, fetchColor is not longer in util but in DbSet.